backstop/tests/slider.html

```diff
@@ -0,0 +1,590 @@
+              data-range-max="100"
+            ></div>
+            <output
+              class="iui-tooltip"
+              style="left: 20%;"
+            >
+              20
+            </output>
+            <div
+              style="left: 80%;"
+              class="iui-slider-thumb"
+              role="slider"
+              aria-label="End Value"
+              aria-valuemin="21"
+              aria-valuenow="80"
+              aria-valuemax="100"
+              data-range-min="0"
+            ></div>
+            <output
+              class="iui-tooltip"
+              style="left: 80%;"
+            >
+              80
+            </output>
+            <div class="iui-slider-ticks">
+              <!-- You could generate the ticks based on your min, max & step values. -->
+              <span class="iui-slider-tick">0</span>
+              <span class="iui-slider-tick">10</span>
+              <span class="iui-slider-tick">20</span>
+              <span class="iui-slider-tick">30</span>
+              <span class="iui-slider-tick">40</span>
+              <span class="iui-slider-tick">50</span>
+              <span class="iui-slider-tick">60</span>
+              <span class="iui-slider-tick">70</span>
+              <span class="iui-slider-tick">80</span>
+              <span class="iui-slider-tick">90</span>
+              <span class="iui-slider-tick">100</span>
+            </div>
+          </div>
+          <span class="iui-slider-max">100</span>
+        </div>
+
+      </div>
+    </section>
+
+    <script>
+
+      function setTooltip(percent, tooltip) {
+        if (tooltip) {
+          tooltip.style.left = `${percent}%`;
+          tooltip.innerHTML = `${percent}`;
+        }
+      }
+
+      function getMinValueFromThumb(thumb) {
+        const minAtt = thumb.getAttribute("aria-valuemin");
+        const dataRangeMinAtt = thumb.getAttribute("data-range-min");
+        const min = parseInt(null !== dataRangeMinAtt ? dataRangeMinAtt : minAtt);
+        return min;
+      }
+
+      function getMaxValueFromThumb(thumb) {
+        const maxAtt = thumb.getAttribute("aria-valuemax");
+        const dataRangeMaxAtt = thumb.getAttribute("data-range-max");
+        const max = parseInt(null !== dataRangeMaxAtt ? dataRangeMaxAtt : maxAtt);
+        return max;
+      }
+
+      function getMaxLimitValueFromThumb(thumb) {
+        var maxAtt = thumb.getAttribute("aria-valuemax");
+        var max = parseInt(maxAtt);
+        return max;
+      }
+
+      function getMinLimitValueFromThumb(thumb) {
+        var minAtt = thumb.getAttribute("aria-valuemin");
+        var min = parseInt(minAtt);
+        return min;
+      }
+
+      function getPercentFromThumb(thumb) {
+        var valueAtt = thumb.getAttribute("aria-valuenow");
+        var value = parseInt(valueAtt);
+        var min = getMinValueFromThumb(thumb);
+        var max = getMaxValueFromThumb(thumb);
+        var percent = ((value - min) / (max - min)) * 100;
```

Comment:
I think this might not be accurate enough for unbound sliders (like the volume one). When you slide the thumb, you see "jitters". Maybe this could be fixed through some animation though 🤔

See continuous sliders at mui for better behavior: https://material-ui.com/components/slider/#continuous-sliders

Reply:
gif of issue: 
![jitter](https://user-images.githubusercontent.com/9084735/125139348-5e09e380-e0de-11eb-9b0b-edf4e738e7d3.gif)

Reply:
There is no jittering in my react implementation. If you want the native js based control to work smoother I will need to update the js. I thought the backstop was mostly to verify css not the js logic.

Reply:
Oh that's good then! I was assuming react would have a similar JS implementation.